op-node/Makefile

```diff
@@ -25,8 +25,14 @@ ifeq ($(shell uname),Darwin)
 	FUZZLDFLAGS := -ldflags=-extldflags=-Wl,-ld_classic
 endif
 
+# Check if TARGETARCH is set to arm64 and adjust compilers accordingly
+ifeq ($(TARGETARCH),arm64)
+    CC := aarch64-linux-gnu-gcc
+    CXX := aarch64-linux-gnu-g++
+endif
```

Comment:
This conditional logic exists in the makefiles instead of the higher level dockerfile. Using if/else to set an `ARG` or `ENV` within a `RUN` statement in the dockerfile changes the values on that docker layer, but the altered values are not recognized by subsequent docker layers. The downside putting the logic in the makefiles is that the same code is duplicated across all relevant makefiles

Reply:
No longer relevant due to decision to use CGO_ENABLED=0 instead of CC and CXX